Makefile

```
@@ -0,0 +1,32 @@
+.DEFAULT_GOAL := help
+
+.PHONY: help build test gen gen-basic gen-custom fuzz-fiql
+
+help:
```

checkmake (0.2.2)

[warning] Target body for "help" exceeds allowed length of 5 (8).

(maxbodylength)